src/helpers/user.helpers.js

```diff
@@ -0,0 +1,24 @@
+const userService = require('../services/user.service');
+
+const nameCheck = (name, res) => {
+  if (!name || typeof name !== 'string') {
+    res
+      .status(400)
+      .send('Invalid request: "name" is required and must be a string.');
+
+    return true;
+  }
+};
+
+const isUserExist = (id, res) => {
+  if (!userService.getById(id)) {
+    res.status(404).send('User with this id not found');
```

Comment:
- Fix them all.
- And choose one style for your function, as some of them are arrow functions and some are declared with the `function` keyword, for example, `getRandomNumberFromUUID`.